Firebase/Core/FIRComponentContainer.m

```diff
@@ -41,13 +41,13 @@ @implementation FIRComponentContainer
 
 #pragma mark - Public Registration
 
-+ (void)registerAsComponentRegistrant:(Class)klass {
++ (void)registerAsComponentRegistrant:(id<FIRLibrary>)klass {
```

Comment:
This should still be `Class<FIRLibrary>` since it'll be the class being passed in, not an instance of the class. Then the `(Class)klass` cast can go away, too.